### ISSUE TITLE
Open directory with Czkawka

### ISSUE THREAD

**Issue by minterior** [qarmin/czkawka#684]
I use Czkawka 4.0.0 in Kubuntu Linux.

I would like to open a directory with Czkawka directly from the file manager (Dolphin in my case) using the context menu. By doing so, it seems the program tries to launch but it does nothing.

Thank you.

**Comment by Sadi58** [comment]
@minterior This is a good idea, and having tried it with the latest linux_czkawka_gui.AppImage (version 4.1.0) under the latest KDE neon with terminal command ~/.local/share/AppImages/linux_czkawka_gui.AppImage ~/Downloads, I can see that it opens with the input folder selected for scanning. It can even be configured to appear in the context menu for folders (Open Folder With) with a little effort.

**Comment by minterior** [comment]
Oh yeah! I've tried it now with Czkawka 4.1.0 installed via snap in Dolphin KDE and it works as expected. The only thing is that when two directories are selected in Dolphin, it opens two different instances of Czkawka instead of just one with the two in the list. Any idea on how to achieve it?

A couple of examples that work like this are Gwenview (image viewer), which only opens a single instance of the application with the images selected, or VLC (media player) opening several videos in the same instance.

Thank you!

**Comment by Sadi58** [comment]
Yes, same here. This needs further input from the developer I think. Maybe an option to allow only one instance? Or simply accepting any number of folders as a parameter?

**Comment by fabiorzfreitas** [comment]
Hi, I just came by to look for this feature!

As a suggestion on how to implement it, perhaps it would be simpler to have this implemented in the CLI and also provide it a feature to call the GUI.

After that, the integration with any OS might be done by simply passing the proper command to the CLI on the OS' context menu!
